nomad/volumewatcher/batcher.go

```diff
@@ -0,0 +1,125 @@
+package volumewatcher
```

Comment:
Note for review: this file draws heavily on https://github.com/hashicorp/nomad/blob/master/nomad/deploymentwatcher/batcher.go